From 12b705ad9bcbd06f33590f3a7685c613c516d742 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 1 Sep 2023 15:35:34 +0200
Subject: [PATCH 0481/1141] dt-bindings: rng: introduce st,stm32mp25-rng
 compatible

Add RNG STM32MP25x platforms compatible.

It supports RNG configuration lock.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I60c2fff5674cf816a0ae380cb9dd24fba6964dfa
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/325019
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 Documentation/devicetree/bindings/rng/st,stm32-rng.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml b/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
index 0055f14a8e3f..591af796a5eb 100644
--- a/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
+++ b/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
@@ -18,6 +18,7 @@ properties:
     enum:
       - st,stm32-rng
       - st,stm32mp13-rng
+      - st,stm32mp25-rng
 
   reg:
     maxItems: 1
@@ -44,6 +45,7 @@ allOf:
           contains:
             enum:
               - st,stm32mp13-rng
+              - st,stm32mp25-rng
     then:
       properties:
         st,rng-lock-conf:
-- 
2.39.2

